src/processHistorian.py

```diff
@@ -85,7 +83,6 @@ def __init__(self):
 
         # Fifth step: Create the OPC UA client
         self._opcua_client = OPCClient(self.__opcua_conf, self._buffer.append)
-        self._opcua_client.connect()
```

Comment:
This does not work. 
connect() also initializes the lists for polling and subscribing. Without this the polling threads will not be created.

Maybe opcuaClient.Client.__init_lists should be edited so no connection is necessary?

Reply:
connect is called by the defibrillator. As namespace ids may have changed during restart of opc reinitializing lists on (re)connect is a good thing IMO

Reply:
Oh I see

Reply:
Thats not the point. 
It is absolutely necessary to initialize the dictionaries in this code segment as the workthreads are based on the list of keys in the polling dictionary. Else the list of keys would be empty and no Threads will be created

Reply:
We cant reliably do that without an ensured connection though. Maybe store the push thread separately (there is only one) and recreate the opc threads from scratch on reconnect.
Sidenote: what happenes, if a node to be polled doesn't exist?

Reply:
This should (tm) now work reliably, given no thread runs away with the lock